tests/core/resource_endpoints_test.py

```diff
@@ -422,3 +424,82 @@ async def test_endpoint(self, mock_utcnow):
         response = await self.test_client.get("/api/hello/world")
         self.assertEqual(response.status_code, 200)
         self.assertEqual(await response.get_json(), {"hello": "world"})
+
+
+class PrivilegedResourceEndpointsTestCase(AsyncFlaskTestCase):
+    use_default_apps = True
+    app_config = {
+        "MODULES": [
+            "tests.core.modules.users",
+            "tests.core.modules.module_with_privileges",
+        ]
+    }
+
+    async def asyncSetUp(self):
```

Comment:
This seems to have no effect, and can be removed from this class

Reply:
leftover. Fixed.